src/app/frontend/commons/commons_module.js

```diff
@@ -0,0 +1,11 @@
+import errorDialogService from './errordialog_service';
+
+/**
+ * Angular module containing navigation chrome for the application.
+ */
+export default angular.module(
+                          'kubernetesDashboard.commons',
+                          [
+                            'ngMaterial',
+                          ])
+    .factory('errorDialog', errorDialogService);
```

Comment:
I see. You can safely disable this lint check. I checked the rule and it does not apply to our codebase: https://github.com/johnpapa/angular-styleguide#style-y040

We're using ES6 classes everywhere so the lint rule adds confusion rather than any benefits.